automation/src/test/scala/org/broadinstitute/dsde/firecloud/fixture/WorkspaceData.scala

```diff
@@ -5,4 +5,9 @@ object WorkspaceData {
   val tags = Map(
     "tag:tags" -> Seq("testing","diabetes")
   )
+
+  val tagAC = Map(
+    "tag:tags" -> Seq("aaaTag", "cccTag")
```

Comment:
recommend using something unique so that this will never fail due to parallel tests or in case someone added this tag to another published workspace.

Reply:
moved tagAC to inside of test. so it's not used by other tests.